### PR TITLE
enterprise contract enhancements

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/HAC-3760
https://issues.redhat.com/browse/RHTAPBUGS-172

## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image

**Click area for expand icon**
image

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by MariaLeonova** [comment]
Overall looks good, made a few suggestion / questions in Slack.

**Comment by MariaLeonova** [comment]
Let's update description text to reference [EC documentation](https://enterprisecontract.dev/docs/ec-policies/index.html), and not any specific Policy. It could say:

> The Enterprise Contract is a set of tools for verifing the provenance of container images built in Red Hat Trusted Application Pipeline and validating them against a clearly defined policy.

The Enterprise Contract policy is defined using the [rego policy language](https://www.openpolicyagent.org/docs/latest/policy-language/) and is described here in [Enterprise Contract Policies](https://enterprisecontract.dev/docs/ec-policies/index.html).

File: src/components/EnterpriseContractView/SecurityEnterpriseContractTab.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.security-enterprise-contract-tab {
```
Comment (rottencandy) [inline src/components/EnterpriseContractView/SecurityEnterpriseContractTab.scss:1]: This doesn't seem to be used anywhere

Comment (sahil143) [reply src/components/EnterpriseContractView/SecurityEnterpriseContractTab.scss]: removed file

**Review by rottencandy — APPROVED** [review]
/lgtm

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**